### ISSUE TITLE
Uncaught ReferenceError: jQuery is not defined - line 23 of append-form-field-element

### ISSUE THREAD

**Issue by honestbleeps** [sonata-project/SonataAdminBundle#868]
I'm running into an error with SonataAdminBundle when I try and add a category (a one to many relationship) by clicking the + button that the Admin Bundle generates.

It pauses for a moment, making an XMLHTTPRequest, and then comes back with "Uncaught ReferenceError: jQuery is not defined"

When viewing the source of the page that came back via that request, it spits out a script tag calling jQuery.  I can't even really understand why it doesn't work - because jQuery is in fact loaded (the web assets are installed properly, and drag/sort even works!)...

The offending line of code is below:

```
<script type="text/javascript">
                jQuery('div#field_container_s4ffee91093cf8_designCategories tbody.sonata-ba-tbody').sortable({
                    axis: 'y',
                    opacity: 0.6,
                    items: 'tr',
                    stop: apply_position_value_s4ffee91093cf8_designCategories
                });
```

   ...

I did just re-run bin/vendors install to ensure I'm on the latest version for my branch.  I'm using Symfony 2.0 and the following deps:

[SonataAdminBundle]
    git=git://github.com/sonata-project/SonataAdminBundle.git
    target=/bundles/Sonata/AdminBundle
    version=origin/2.0

[SonataDoctrineORMAdminBundle]
   git=https://github.com/sonata-project/SonataDoctrineORMAdminBundle
   target=bundles/Sonata/DoctrineORMAdminBundle
   version=origin/2.0

**Comment by eXtreme** [comment]
It shouldn't be closed as it is still an issue.

**Comment by krewetka** [comment]
:+1: 

have the same issue with sonata_type_collection AND a file type in the some form

**Comment by krewetka** [comment]
I can confirm that using file from https://github.com/sonata-project/SonataAdminBundle/commit/09adf24dfbf0ddbd79f427558f478c6367f1659d is fixing in in my sonata admin 2.3

I hate modifying vendor code but as far as I see there is no other option as this probably never get fixed in 2.3 and 2.4 is still using 2.64 version from 2011.

**Comment by m79reed** [comment]
I'm using sonata-project/admin-bundle 2.2.x-dev (jquery.form.js v2.64). Got this error when using "file" type in form. With jquery.form.js version: 3.45.0-2013.10.17 works fine.

Thanks @krewetka.
